etc/web/aliases-py26.conf

```diff
@@ -1,23 +1,23 @@
-WSGIScriptAlias /ping                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/ping.py
-WSGIScriptAlias /redirect                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/redirect.py
-WSGIScriptAlias /replicas                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/replica.py
-WSGIScriptAlias /requests                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/request.py
-WSGIScriptAlias /rses                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/rse.py
-WSGIScriptAlias /rules                   /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/rule.py
-WSGIScriptAlias /scopes                  /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/scope.py
-WSGIScriptAlias /subscriptions           /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/subscription.py
-WSGIScriptAlias /traces                  /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/trace.py
-WSGIScriptAlias /objectstores            /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/objectstore.py
-WSGIScriptAlias /lifetime_exceptions     /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/lifetime_exception.py
+WSGIScriptAlias /accounts                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/account.py
+WSGIScriptAlias /accountlimits           /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/account_limit.py
+WSGIScriptAlias /auth                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/authentication.py
+WSGIScriptAlias /config                  /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/config.py
+WSGIScriptAlias /tmp_dids                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/temporary_did.py
+WSGIScriptAlias /dids                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/did.py
+WSGIScriptAlias /identities              /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/identity.py
+WSGIScriptAlias /heartbeats              /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/heartbeat.py
+WSGIScriptAlias /locks                   /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/lock.py
+WSGIScriptAlias /meta                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/meta.py
+WSGIScriptAlias /nongrid_traces          /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/nongrid_trace.py
+WSGIScriptAlias /ping                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/ping.py
+WSGIScriptAlias /redirect                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/redirect.py
+WSGIScriptAlias /replicas                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/webpy/replica.py
```

Comment:
v1 missing in the paths ?

E.g., /rest/webpy/v1/replica.py ?

Reply:
No, that's a typo. It should go directly to `rucio/web/rest/<module>.py`. 
In the main directory then are only links to the correct version. E.g., for the moment `rucio/web/rest/account.py` links to `rucio/web/rest/webpy/v1/account.py`. And then later if we want to start using flask we can just adapt the symlinks.